source/docs/software/basic-programming/java-units.rst

```diff
@@ -68,13 +117,28 @@ Unit conversions can be done by calling ``Measure#in(Unit)``. The Java type syst
      maxAcceleration.in(MetersPerSecondPerSecond)
    );
 
-.. note:: Due to restrictions of the Java type system, acceleration is modeled as ``Velocity<Velocity<X>>>`` instead of having its own class.
+Usage Example
+~~~~~~~~~~~~~
+
+Pulling all of the concepts together, we can create an example that calculates the end effector position of an arm mechanism:
+
+.. code-block:: java
+
+  Measure<Distance> armLength = Feet.of(3).plus(Inches.of(4.25));
+  Measure<Distance> endEffectorX = armLength.times(Math.cos(getArmAngle().in(Radians)));
+  Measure<Distance> endEffectorY = armLength.times(Math.sin(getArmAngle().in(Radians)));
+
+Human-readable Formatting
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``Measure`` class has both ``toShortString()`` and ``toLongString()`` methods that can be used to get a human-readable representation of the measure. This feature is useful to display a measurement on a dashboard or in logs.
```

Comment:
It might be helpful to add an example which shows the difference between `toShortString` and `toLongString`.

Reply:
Is there rationale for not overriding `toString()` that should be added to this section? That seems like an oversight, but maybe there was a reason.